codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/awssdk/shapevisitor/ShapeVisitorHelper.java

```diff
@@ -13,11 +14,19 @@
 
 public class ShapeVisitorHelper {
 
-  public static final Map<MemberShape, Boolean> TO_DAFNY_OPTIONALITY_MAP =
+  private static final Map<MemberShape, Boolean> optionalShapesToDafny =
```

Comment:
Shouldn't this be a in snake case? 

optionalShapesToDafny => OPTIONAL_SHAPES_TO_DAFNY

Reply:
Not necessarily - this isn't public anymore.